app/steps/ui/executors/address/index.js

```diff
@@ -54,6 +56,12 @@ class ExecutorAddress extends AddressStep {
         if (ctx.index === -1) {
             ctx.allExecsApplying = ctx.executorsWrapper.areAllAliveExecutorsApplying();
         }
+        if (ctx.addresses && !session.addresses.executors) {
+            session.addresses.executors = [];
+        }
+        if (ctx.addresses) {
+            session.addresses.executors.push(ctx.addresses);
+        }
```

Comment:
Can we have unit tests to test this logic?

Reply:
ah yes that is the 'returns the correct data and errors' in the handlePost() in testExecutorAddress